.storybook/main.js

```diff
@@ -3,53 +3,18 @@
  * Licensed under the MIT License.
  */
 
-const SourcePlugin = require('./sourcecode-addon/webpackPlugin');
-const path = require('path');
-
 module.exports = {
   stories: ['../src/**/*.stories.*'],
   addons: [
     '@storybook/preset-create-react-app',
     '@storybook/addon-docs', // Enables MDX
     './sourcecode-addon/register'
   ],
-  webpackFinal: async (config) => {
-    // Used by sourcecode-addon. Add loader that registers raw source code in a cache and display it in the Source tab.'
-    config.module.rules.push({
-      test: /\.jsx$|\.tsx$|\.css$|\.ts$|\.js$|\.mdx$/,
-      use: [
-        {
-          loader: path.resolve(__dirname, 'sourcecode-addon/webpackLoader.js'),
-          options: { root: path.resolve(__dirname, '../src') },
-        },
-      ],
-    });
```

Comment:
Don't you still need these additional rules for building the stories?

Reply:
No - those are just for building up the rawSources.json file. Since that's move into a separate script now it can go.